pubspec.yaml

```
@@ -7,7 +7,7 @@ issue_tracker: https://github.com/100mslive/100ms-flutter/issues
 documentation: https://www.100ms.live/docs
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.16.0 <4.0.0"
   flutter: ">=1.20.0"
```

Flutter version to be updated as well
```suggestion
  flutter: ">=2.10.0"
```